src/actions/config.js

```diff
@@ -1,15 +1,15 @@
 import yaml from "js-yaml";
-import { set, defaultsDeep, get } from "lodash";
+import { set, defaultsDeep, get, includes } from "lodash";
 import { authenticateUser } from "Actions/auth";
 import * as publishModes from "Constants/publishModes";
 
 export const CONFIG_REQUEST = "CONFIG_REQUEST";
 export const CONFIG_SUCCESS = "CONFIG_SUCCESS";
 export const CONFIG_FAILURE = "CONFIG_FAILURE";
 
-const validTypes = [ "text/yaml", "application/x-yaml" ];
+const validTypes = { TEXT_YAML: "text/yaml", APPLICATION_X_YAML: "application/x-yaml" };
```

Comment:
Why not just use an array of types?

Reply:
@tech4him1 I was looking through other parts code and saw types as an object, in `ListControl.js` , so I figured I'd follow in previous footsteps. I can revert back to an array, if you'd like?

Reply:
Let me think a little about how this would best work with our `formats.js` for supporting multiple config formats in the future.